### PR TITLE
Add Suroi.io to Games

### LINKED ISSUE #323: Add Suroi.io
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

Please fill out information below (all fields are mandatory unless noted otherwise):

```yaml
# software name
name: "Suroi"
# URL of the software project's homepage
website_url: "https://suroi.io"
# URL where the full source code of the program can be downloaded
source_code_url: "https://github.com/HasangerGames/suroi"
# description of what the software does, shorter than 250 characters, sentence case
description: "An open source video game, inspired by the legendary surviv.io game, that can also be self-hosted. This open source game was created after surviv.io unexpectedly decided to shut down operations and thousands of players left with no alternatives."
# list of license identifiers, see https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/licenses.yml for the full list of licenses
licenses:
  - GNU GPL-3.0
# list of languages/platforms, see https://github.com/awesome-selfhosted/awesome-selfhosted-data/tree/master/platforms for the full list of platforms
platforms:
  - Javascript
  - HTML
  - Nodejs
  - Typescript
# list of tags (categories), see https://github.com/awesome-selfhosted/awesome-selfhosted-data/tree/master/tags for the full list of tags
tags:
  -  games
# (optional, yes/no, default no) whether the software depends on a third-party service outside the user's control
depends_3rdparty: no
# (optional) link to an interactive demo of the software
demo_url: "https://suroi.io/"
# (optional) link to a list of clients/addons/plugins/apps/bots... for the software
```

To ensure your issue is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x ] Submit one item per pull issue. This eases reviewing and speeds up inclusion.
- [ x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.

### PR DESCRIPTION
**Pull request by kokomo123**
This should close #323 as fulfilling the requirements. 

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [X] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [X] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [X] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [X] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [X] Comments and unused optional fields have been removed.
- [X] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [X] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [X] Any software project you are adding to the list is actively maintained.
- [X] Any software project you are adding was first released more than 4 months ago.
- [X] Any software project you are adding has working installation instructions.
- [ ] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [X] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/suroi.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: "Suroi"
+website_url: "https://suroi.io/"
+source_code_url: "https://github.com/HasangerGames/suroi"
+description: "An open source video game, inspired by the legendary surviv.io game, that can also be self-hosted. This open source game was created after surviv.io unexpectedly decided to shut down operations and thousands of players left with no alternatives."
```
Comment (Ki-er) [inline software/suroi.yml:4]:
```suggestion
description: "Clone of the now abandoned, Surviv.io"
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
A cleaner description.

**Comment by kokomo123** [comment]
Thank you for the adjustment of the description! I've committed the changes.

File: software/suroi.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: "Suroi"
+website_url: "https://suroi.io/"
+source_code_url: "https://github.com/HasangerGames/suroi"
+description: "Clone of the now abandoned, Surviv.io."
```
Comment (nodiscc) [inline software/suroi.yml:4]:
```suggestion
description: "An open-source 2D battle royale game inspired by surviv.io."
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]
Thanks

**Pull request merged**